lib/LaTeXML/Package.pm

```diff
@@ -899,17 +899,31 @@ sub generateID_nextid {
 #
 #======================================================================
 
-# Return $tokens with all tokens expanded
+# Return $tokens with all tokens fully expanded
+our $T_EXPAND_END_CHECK = T_OTHER('END EXPANSION');
```

Comment:
Shouldn't that be a `T_MARKER` ?

Reply:
Well, yeah, but `T_MARKER` never make it out of `Gullet`. OTOH, perhaps if there's a helper function from within `Gullet`, something safer could be arranged? Either way, it's all about error recovery.

Reply:
Right, but if the `T_OTHER` makes it out of Gullet and encounters some processing in the Error realm, it may make it out into the final XML. Which we _may_ appreciate, but I suspect the average user won't...

That said, the last time I used a magic token I invented my own catcode (`CC_SMUGGLE_THE`), so I don't really have a clear head on the topic. Your call, it works as-is.